lib/shared/widgets/default_button/default_button.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:hermes_app/shared/theme/app_colors.dart';
+import 'package:hermes_app/shared/widgets/default_button/utils/dafault_button_mixin.dart';
+
+class DefaultButton extends StatelessWidget with DefaultButtonMixin {
+  final VoidCallback onPressed;
+  final Widget title;
+  final bool isLoading;
+  final bool enabled;
+  const DefaultButton({
+    super.key,
+    required this.onPressed,
+    required this.title,
+    this.isLoading = false,
+    this.enabled = true,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 45,
+      width: double.maxFinite,
```

Comment:
Aqui o ideal seria usar a extensão do nosso app
```suggestion
      width: context.screenWidth,
```